cns/middlewares/k8sSwiftV2.go

```diff
@@ -249,3 +249,59 @@ func (k *K8sSWIFTv2Middleware) getIPConfig(ctx context.Context, podInfo cns.PodI
 func (k *K8sSWIFTv2Middleware) Type() cns.SWIFTV2Mode {
 	return cns.K8sSWIFTV2
 }
+
+// CNS gets pod CIDRs from configuration env and parse them to get the v4 and v6 IPs
+// Containerd reassigns the IP to the adapter and kernel configures the pod cidr route by default, so windows swiftv2 does not require pod cidr
```

Comment:
I feel like the comment here would be better above the OS-specific implementations of getInfraRoutes. GetPodCidrs (the function) by _itself_ doesn't do anything os-specific.